skin/frontend/base/default/js/oyst/oneclick.js

```diff
@@ -163,3 +164,24 @@ function isOystOneClickButtonFormValid(formName) {
 function prependChild(parentElement, newFirstChildElement) {
     parentElement.insertBefore(newFirstChildElement, parentElement.firstChild);
 }
+
+function configurableProductCustomOptions() {
+    var customOptions = $$('.product-custom-option'),
```

Comment:
Use VanillaJs (document.getElementsby... ) to me more compatible with customer front and in use double quote ".